e2e-test/cypress/integration/documents_spec.js

```diff
@@ -71,33 +81,46 @@ describe("Attaching a document to a workflowitem.", function() {
     });
 
     // make sure the validation was successful:
-    cy.get("[data-test=workflowitem-documents-tab]").click();
+    cy.get("[data-test=workflowitem-documents-tab]")
+      .should("be.visible")
+      .click();
     cy.get(`button[label="Validated!"] > span`).should("contain", "OK");
   });
 
   it("Validation of wrong document fails.", function() {
     const fileDisplayName = "Wrong_Document_Test";
     uploadDocument(fileDisplayName, fileName);
     // submit and close the dialog:
-    cy.get("[data-test=submit]").click();
+    cy.get("[data-test=submit]")
+      .should("be.visible")
+      .click();
 
```

Comment:
wait for subproject.viewdetails before opening the info dialog. If it is too fast the document isn't shown in the documents tab.

Reply:
route and wait is now set